test/install_test.go

```diff
@@ -417,7 +417,7 @@ func TestCheckHelmStableBeforeUpgrade(t *testing.T) {
 		t.Fatalf("linkerd-smi-metrics SA labeling failed: %s", err)
 	}
 
-	testCheckCommand(t, "", TestHelper.UpgradeHelmFromVersion(), "", TestHelper.UpgradeHelmFromVersion())
+	testCheckCommand(t, "", TestHelper.UpgradeHelmFromVersion(), "", TestHelper.UpgradeHelmFromVersion(), false)
```

Comment:
Can you add a TODO comment explaining this is a temporary measure because 2.7.1 didn't have the `linkerd-addons` check section?

Reply:
I can definitely add a TODO, but it makes for `TestCheckHelmStableBeforeUpgrade` to not check for the output, as its a before thing and checks could change in the new CLI.

Reply:
Correct, but after 2.8 comes out, we should test that upgrading from 2.8 to whatever edge/stable comes next has a sane `linkerd check` output, thus not requiring skipping that anymore.
That being said, it's normal for this output to change from version to version, so a better long-term solution would be to have specific per-version outputs, but that'd be something for another PR :-)